xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -325,7 +334,7 @@ bool CGUIFontTTF::Load(
   m_posY = -(int)GetTextureLineHeight();
 
   // cache the ellipses width
-  Character *ellipse = GetCharacter(L'.');
+  Character* ellipse = GetCharacter(L'.', NULL);
```

Comment:
Second argument is _unsigned int_ so NULL is not appropriate - use 0?
And instead of _unsigned int_ you could use FT_UInt.

Reply:
Done